### PR TITLE
[ASR][A2A] Adding support for proximity placement group

### PR DESCRIPTION
**Pull request by ayfathim**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2018-07-10/service.json [hunk 1]
```diff
@@ -7497,6 +7497,10 @@
           "description": "The recovery availability set Id.",
           "type": "string"
         },
+        "recoveryProximityPlacementGroupId": {
+          "description": "The recovery proximity placement group Id.",
+          "type": "string"
+        },
```
Comment (fengzhou-msft) [inline specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2018-07-10/service.json]: Add this property in example as well?

Comment (ayfathim) [reply specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2018-07-10/service.json]: Azure Site Recovery (ASR) service supports multiple replication providers and following are published providers (under ReplicationProviderSpecificSettings): * A2AReplicationDetails * HyperVReplicaAzureReplicationDetails * HyperVReplicaBlueReplicationDetails * HyperVReplicaReplicationDetails * InMageAzureV2ReplicationDetails * InMageReplicationDetails
All the resource operations examples are with one such provider "HyperVReplicaAzure".
And the new properties belong to "A2A".
We cannot update the examples.

**Comment by ayfathim** [comment]
@fengzhou-msft : please include the required people from ARM team to review this.

**Comment by ayfathim** [comment]
@Azure/arm-api-review-board : Please review this

**Review by KrisBash — APPROVED** [review]
LGTM

**Review by fengzhou-msft — APPROVED** [review]

**Pull request merged**